src/harness/fake_sas.py

```diff
@@ -307,6 +313,8 @@ def do_POST(self):
       response = FakeSas().Relinquishment(request)
     elif self.path == '/%s/deregistration' % self.version:
       response = FakeSas().Deregistration(request)
+    elif self.path == '/admin/injectdata/ex_zone':
+      response = FakeSasAdmin().InjectExZone(request)
```

Comment:
Any reason this can't be part of the large "elif" on line 320?

Reply:
Moved the InjectExclusionZone to the elif part